rust/cargo-psibase/src/link.rs

```diff
@@ -55,7 +55,7 @@ fn validate_polyfill(polyfill: &walrus::Module) -> Result<(), anyhow::Error> {
         return Err(anyhow!("Polyfill has unexpected section"));
     }
     if polyfill.types.iter().next().is_none()
-        || polyfill.imports.iter().next().is_none()
+        // || polyfill.imports.iter().next().is_none() // Not required for tester
```

Comment:
The polyfills are needed for the services wasms still. This link code works for both.

Reply:
@sparkplug0025  you already skipped everything in here now that the polyfill is gone:
``` 
if polyfill_source_module.funcs.iter().next().is_some() {
        pretty_path("Polyfilling", filename);
        link_module(&polyfill_source_module, &mut dest_module)?;
    } else {
        pretty("Polyfilling", "SKIPPED! No polyfill functions found");
    }
```

Reply:
ah right, so I dont need to care about imports checks. I was just worried if we reintroduce some polyfilling that does not need any imports. 

I can remove it!

Reply:
I guess it is a bit of a weird requirement that a polyfill must have imports...
So I would say this change is unrelated to the rest of your PR, but that's okay. If we want to keep it then delete it rather than commenting it out